build.bash

```diff
@@ -37,14 +28,27 @@ for i in `ls $SCRIPT_DIR/src | grep -v vendor`
 do
 	echo $i
 	go install $i
+	if [[ $? != 0 ]]; then
+	    LINUX_PACKAGES_ERRORS=true
```

Comment:
When tweaking a dependency, we can break multiple tools. It seems better to abort the build on the first error (even though this is not the existing behavior).

Thoughts? @djfritz